libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1650,7 +1636,7 @@ void AP_InertialSensor::wait_for_sample(void)
     }
 
 check_sample:
-    if (!_hil_mode) {
+    if (true) {
```

Comment:
better to remove if

Reply:
Removed it but didn't reindent the very substantial block